components/script/dom/fontface.rs

```diff
@@ -540,6 +540,9 @@ impl FontFaceMethods<crate::DomTypeHolder> for FontFace {
         )
         .expect("Parsing shouldn't fail as descriptors are valid by construction");
 
+        // Construct a WebFontDocumentContext object for the current document.
```

Comment:
I don't think this comment (and the similar ones in the rest of the PR) give readers any new information. Let's remove them.